.eslintrc.js

```diff
@@ -12,6 +12,12 @@ module.exports = {
         "import/no-extraneous-dependencies": [0, { devDependencies: false }],
       },
     },
+    {
+      files: ["*/**/ducks/*.ts"],
+      rules: {
+        "no-param-reassign": "off",
+      },
+    },
```

Comment:
this is something we discussed in Slack a while back. Disabling this rule allows us to use immer in reducers